### PR TITLE
Text section for teams

### PR DESCRIPTION
**Pull request by KevinCawley**
Adds the text for the teams section of the website
**Description**

Does all of the text, but leaves the grid and tables. 
**Motivation and context**

**How has this been tested?**
- [ ] Testing pipeline.
- [x] Other. 

**Examples**

**Type of change**

- [ ] Bug fix. 
- [x] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [ ] My change requires a change to the documentation.
    - [ ] I have updated the documentation accordingly.
    - [ ] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [ ] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

**Comment by KevinCawley** [comment]
https://kevincawley.github.io/tardis-sn.github.io/branch/teams_page/

File: content/team/community_roles.md [hunk 1]
```diff
@@ -4,4 +4,117 @@ date: 2021-06-28T13:44:37-05:00
...
+#### TARDIS Core
+ - Team that oversees the activities and sets the goals for the TARDIS Collaboration
+ - Identifies science opportunities that match TARDIS capabilities or coordinates the addition of capabilities
+ - Identifies education opportunities such as workshops, lecture development, and outreach opportunities
+
+#### TARDIS GSoC Coordinators (2020)
+ - Coordinate participation of TARDIS in the Google Summer of Code
+
+#### TARDIS GSoC Mentors (2020)
+ - Mentors for Google Summer of Code students
```
Comment (jaladh-singhal) [inline content/team/community_roles.md:49]: Remove them coz we have separate section for them below

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]

File: content/team/collaboration.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+| GSoC 2016  | Karan Desai   | <a href="https://summerofcode.withgoogle.com/archive/2016/projects/4796129849901056/" target="_blank" rel="noopener nofollow"> Extensive Test Suite for TARDIS<a>   |
+| GSoC 2016  | Mikhail Mishin   | <a href="https://summerofcode.withgoogle.com/archive/2016/projects/6699840280985600/" target="_blank" rel="noopener nofollow">Carsus - TARDIS support package for creating atomic datasets.</a>   |
+| GSoC 2016  | Fotis Tsamis  | <a href="https://summerofcode.withgoogle.com/archive/2016/projects/5656164301799424/" target="_blank" rel="noopener nofollow">Improvement of the TARDIS configuration system</a>  |
+| GSoC 2015  | Aoife Boyle   | <a href="https://www.google-melange.com/archive/gsoc/2015/orgs/tardis_sn/projects/aboyle.html" target="_blank" rel="noopener nofollow">Implementation of new features into the new Tardis plasma module framework.</a>   |
+| GSoC 2015  | Vytautas Jančauskas   | <a href="https://www.google-melange.com/archive/gsoc/2015/orgs/tardis_sn/projects/bucket_brigade.html" target="_blank" rel="noopener nofollow">TARDIS-SN Fitting Multiple Supernova Spectra at Once</a>   |
+| GSoC 2014  | Vytautas Jančauskas   | No Report   |
```
Comment (jaladh-singhal) [inline content/team/collaboration.md:30]: It was under PSF - https://www.google-melange.com/archive/gsoc/2014/orgs/python/projects/bucket_brigade.html

File: content/team/collaboration.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+if not for the work of the following active and past members 
+of our community. To know more about members with specific 
+roles, visit the <a href="../community_roles/">Community Roles</a> page.
+
+## GSoC/SoCIS Students
+A brief explanation on GSoC & SoCIS
+
+| Program  | Name     | Project  |
+|----------|----------|----------|
+| GSoC 2020  | Jordi Eguren Brown   | <a href="https://summerofcode.withgoogle.com/archive/2020/projects/5699342318436352/" target="_blank" rel="noopener nofollow">TARDIS Code reestructure with factory method</a>   |
```
Comment (jaladh-singhal) [inline content/team/collaboration.md:17]:
```suggestion
| GSoC 2020  | Jordi Eguren Brown   | <a href="https://summerofcode.withgoogle.com/archive/2020/projects/5699342318436352/" target="_blank" rel="noopener nofollow">TARDIS Code restructure with factory method</a>   |
```

File: content/team/collaboration.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+draft: false
+layout: page
+---
+The present version of TARDIS would not have been possible 
+if not for the work of the following active and past members 
+of our community. To know more about members with specific 
+roles, visit the <a href="../community_roles/">Community Roles</a> page.
+
+## GSoC/SoCIS Students
+A brief explanation on GSoC & SoCIS
```
Comment (jaladh-singhal) [inline content/team/collaboration.md:13]: Can you please put a brief explaination of what GSoC and SoCIS are (hyperlinking them to their website)?

File: content/team/community_roles.md [hunk 5]
```diff
@@ -4,4 +4,150 @@ date: 2021-06-28T13:44:37-05:00
...
+    </div>
+    <div class ="individual-container">
+        <div class ="role-box">Principal Investigator</div>
+        <div class ="info-container">
+            <div class ="person-name">Brilliant Sunset</div>
+            <img class="rounded-picture" src="../pictures/random-image.jpg"> 
+            <div class ="small-bio">Biography</div>
+        </div>
+    </div>
+</div>
```
Comment (jaladh-singhal) [inline content/team/community_roles.md:67]: Just write "Picture Grid (under construction)"

**Review by jaladh-singhal — CHANGES_REQUESTED** [review]

File: content/team/collaboration.md [hunk 6]
```diff
@@ -10,24 +10,25 @@ of our community. To know more about members with specific
 roles, visit the <a href="../community_roles/">Community Roles</a> page.
 
 ## GSoC/SoCIS Students
-A brief explanation on GSoC & SoCIS
+The Google Summer of Code (GSoc) is a global program focused on introducting students to open source software development. 
+The Summer of Code In Space (SoCIS) is a program that aims at promoting open-source space-related software to students.
```
Comment (jaladh-singhal) [inline content/team/collaboration.md:14]:
```suggestion
The [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/) is a global program organized by Google where students work with an open source organization on a programming project during Summers. [Summer of Code In Space (SoCIS)](https://www.esa.int/Enabling_Support/Space_Engineering_Technology/SOCIS_The_ESA_Summer_of_Code_in_Space) is another similar program by ESA that aims at promoting open-source "space-related" software to students by letting them code new functionalities.
```

File: content/team/community_roles.md [hunk 7]
```diff
@@ -4,4 +4,114 @@ date: 2021-06-28T13:44:37-05:00
...
+### Members & Roles
+Current TARDIS core team members are as follows (you can click on the role to know what it means):
+
+Picture grid of core team members (following is an example of how SunPy does it). It will consist of:
+ - Name
+ - Picture (not Avatar)
+ - Role(s) (linking to their corresponding description section (through permalinks))
+ - Short Bio (can add link in that if people want)
+ 
+ This is in here as a placeholder!
```
Comment (jaladh-singhal) [inline content/team/community_roles.md:32]:
```suggestion
Picture Grid of core team (under construction)
```

**Review by jaladh-singhal — APPROVED** [review]

**Pull request merged**